filebeat/harvester/harvester.go

```diff
@@ -17,12 +17,15 @@ import (
 	"fmt"
 	"regexp"
 
+	"github.com/satori/go.uuid"
+
 	"github.com/elastic/beats/filebeat/config"
 	"github.com/elastic/beats/filebeat/harvester/encoding"
 	"github.com/elastic/beats/filebeat/input"
 )
 
 type Harvester struct {
+	Id                 uuid.UUID
```

Comment:
The thing I don't like about the above is that currently beat.UUID is not accessible from the harvester. Perhaps it is easier to remove the UUID addition from this PR and continue the discussion in https://github.com/elastic/beats/pull/964 (where it is coming from initially) as there it in more context?